lumeer-storage/lumeer-storage-mongodb/src/main/java/io/lumeer/storage/mongodb/dao/project/MongoDocumentDao.java

```diff
@@ -81,6 +81,7 @@ public void deleteDocumentsRepository(final Project project) {
    @Override
    public Document createDocument(final Document document) {
       try {
+         document.setDataVersion(0);
```

Comment:
This is unnecessary, because version is excluded when writing to DB. The version is set only by inc operation.